README.md

```diff
@@ -11,10 +11,14 @@ git clone https://github.com/yaptide/yaptide.git
 ## Building and running the app
 
 There are two main ways of build and run the app
-- [For developers](./docs/for_developers.md), which is setting up all components manually. It is faster then Docker.
-- [Using Docker](./docs/using_docker.md). Used for the deployment, takes more time.
+- [For developers](https://yaptide.github.io/for_developers/backend/for_developers/), which is setting up all components manually. It is faster then Docker.
+- [Using Docker](https://yaptide.github.io/for_developers/backend/using_docker/). Used for the deployment, takes more time.
 
 
+## User documentation
```

Comment:
Move that user documentation to the very top of this README.md, even before `## Getting the code`